SalesforceCommerceCloud/pwa-kit
# Design enhancements for product sets

# Description
This PR polishes up design elements and implements UX enhancements for the new product sets feature. 


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Removal of the 'Complete the Set' carousal for product sets
- Lazy loading of product sets children images
- PDP: Added-to-cart modal scroll and sticky button behaviors in mobile
- PDP: Added-to-cart modal is 90% of viewport and scrollable in desktop

# How to Test-Drive This PR

- Checkout this branch
- Run `npm start`
- Verify that all changes are visible and working correctly in the template (i.e. the children images only load on scroll)

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
